docs/src/use/configure/migration-guide.md

```diff
@@ -580,7 +580,7 @@ In flat config, dotfiles (e.g. `.dotfile.js`) are no longer ignored by default.
 
 Eslintrc files let you configure the linter itself with the `noInlineConfig` and `reportUnusedDisableDirectives` properties.
 
-The flat config system introduces a new top-level property `linterOptions` that you can use to configure the linter. In the `linterOptions` object, you can include `noInlineConfig` and `reportUnusedDisableDirectives`.
+The flat config system introduces a new top-level property `linterOptions` that you can use to configure the linter. In the `linterOptions` object, you can include `noInlineConfig`, `reportUnusedDisableDirectives`, and `reportUnusedInlineConfigs`.
```

Comment:
I'm not sure if it makes sense to mention the new option in the config migration guide because it didn't exist in eslintrc config format.